api/ping.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package docker.swarmkit.v1;
+
+import "gogoproto/gogo.proto";
+import "plugin/plugin.proto";
+
+// Ping is the API provided by managers in the cluster to prove reachability over their
+// advertised address.
+service Ping {
```

Comment:
ooooh, there is even https://github.com/grpc/grpc-go/blob/master/health/health.go